README.md

```diff
@@ -18,52 +18,106 @@ The NTT manager contracts additionally keep track of the token decimals of the o
 
 The payload includes the trimmed amount, together with the decimals that trimmed amount is expressed in. This number is the minimum of (8, source token decimals, destination token decimals).
 
-### NTT Message Lifecycle
+# Rate-Limiting and Cancel Flows
 
-### EVM
-1. **Sending**: A client calls on [`transfer`] to initiate an NTT transfer. The client must specify at minimum, the amount of the transfer, the recipient chain, and the recipient address on the recipient chain. [`transfer`] also supports a flag to specify whether the NTTManager should queue transfers if they are rate-limited. Clients can also include additional instructions to forward along to its peer NTT Manager on the recipient chain to execute. Depending on the mode, transfers are either "locked" or "burned". Once the transfer has been forwarded to the Transceiver, the NTTManager emits the following event:
+Ntt supports rate limiting on the sending. If a trnasfer is rate limited on the source chain and queueing is enabled, transfers are placed into a outbound queue that can be released after a 24-hr window via the [`completeInboundTransferQueue`] method.
+
+Transfers that are rate-limited on the destination chain are added to an InboundTransferQueue that can be released via the [`completeInboundTransferQueue`] method.
```

Comment:
Transfers that are rate-limited on the destination chain are added to an inbound transfer queue. Similar to the outbound queue, these transfers can be released after the configured `_rateLimitDuration` has expired via the [`completeInboundTransferQueue`] method.